core/src/main/java/org/jboss/pnc/build/finder/core/BuildConfig.java

```diff
@@ -96,6 +96,9 @@
     @JsonAlias("use-checksums-file")
     private Boolean useChecksumsFile;
 
+    @JsonAlias("pnc-parallelism-threshold")
+    private Long pncParallelismThreshold;
+
```

Comment:
You also need to add the option to `cli/src/main/java/org/jboss/pnc/build/finder/cli/Main.java`.

Reply:
sure thing